awx/ui_next/src/screens/Inventory/InventoryList/InventoryList.test.jsx

```diff
@@ -140,186 +141,174 @@ describe('<InventoriesList />', () => {
+    expect(items).toHaveLength(3);
+    items.forEach(item => {
+      expect(item.prop('isSelected')).toEqual(true);
+    });
+
+    expect(
+      wrapper
+        .find('InventoryListItem')
+        .first()
+        .prop('isSelected')
+    ).toEqual(true);
   });
 
-  test('delete button is disabled if user does not have delete capabilities on a selected inventory', async done => {
-    const wrapper = mountWithContexts(<InventoriesList />);
-    wrapper.find('InventoriesList').setState({
-      inventories: mockInventories,
-      itemCount: 3,
-      isInitialized: true,
-      selected: mockInventories.slice(0, 2),
+  test('should disable delete button', async () => {
+    let wrapper;
+    await act(async () => {
+      wrapper = mountWithContexts(<InventoryList />);
     });
-    await waitForElement(
-      wrapper,
-      'ToolbarDeleteButton * button',
-      el => el.getDOMNode().disabled === false
-    );
-    wrapper.find('InventoriesList').setState({
-      selected: mockInventories,
+    wrapper.update();
+
+    await act(async () => {
+      wrapper
+        .find('InventoryListItem')
+        .at(2)
+        .invoke('onSelect')();
     });
-    await waitForElement(
-      wrapper,
-      'ToolbarDeleteButton * button',
-      el => el.getDOMNode().disabled === true
+    wrapper.update();
+
+    expect(wrapper.find('ToolbarDeleteButton button').prop('disabled')).toEqual(
+      true
     );
-    done();
   });
 
-  test('api is called to delete inventories for each selected inventory.', () => {
-    InventoriesAPI.destroy = jest.fn();
-    const wrapper = mountWithContexts(<InventoriesList />);
-    wrapper.find('InventoriesList').setState({
-      inventories: mockInventories,
-      itemCount: 3,
-      isInitialized: true,
-      isModalOpen: true,
-      selected: mockInventories.slice(0, 2),
+  test('should call delete api', async () => {
+    let wrapper;
+    await act(async () => {
+      wrapper = mountWithContexts(<InventoryList />);
+    });
+    wrapper.update();
+
+    await act(async () => {
+      wrapper
+        .find('InventoryListItem')
+        .at(0)
+        .invoke('onSelect')();
     });
-    wrapper.find('ToolbarDeleteButton').prop('onDelete')();
+    wrapper.update();
+    await act(async () => {
+      wrapper
+        .find('InventoryListItem')
+        .at(1)
+        .invoke('onSelect')();
+    });
+    wrapper.update();
+    await act(async () => {
+      wrapper.find('ToolbarDeleteButton').invoke('onDelete')();
+    });
+
     expect(InventoriesAPI.destroy).toHaveBeenCalledTimes(2);
   });
 
-  test('error is shown when inventory not successfully deleted from api', async done => {
+  test('should show deletion error', async () => {
     InventoriesAPI.destroy.mockRejectedValue(
       new Error({
         response: {
           config: {
             method: 'delete',
-            url: '/api/v2/inventories/1',
+            url: '/api/v2/teams/1',
```

Comment:
I don't think the actual error response matters here but the url would be `/api/v2/inventories/1` - this is probably some copy/paste (?)